scripts/xpp_update_happi_line

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+"""
+Updates items in the happi database to reflect the which line XPP is 
+using (mono line, pink line).
+
+Assumes pcds-conda, and grabs the currently active happi config.
+"""
+
+import argparse
+
+import click
+import happi
+from ophyd import EpicsSignalRO
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog='xpp_update_happi_line', description=__doc__
+    )
+
+    parser.add_argument('--min', dest='min_z', action='store', type=float,
+                        default=784, required=False, help='min z range')
+
+    parser.add_argument('--max', dest='max_z', action='store', type=float,
+                        default=788, required=False, help='max z range')
+
+    parser.add_argument('--dry-run', action='store_true', dest='dry_run',
+                        default=False, required=False,
+                        help='run script without saving to database')
+
+    args = parser.parse_args()
+    max_z = args.max_z
+    min_z = args.min_z
+    pink_active = EpicsSignalRO('XPP:INS:POS:01:IN_DI').get()
+    mono_active = EpicsSignalRO('XPP:INS:POS:01:OUT_DO').get()
```

Comment:
I think this is a good approach. The only other approach I can think of is having a mechanism for creating happi devices that "move/modify" other happi devices, so then the xpp table can be one such device. That might get complicated quickly and not be worth the hassle.